<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_dialog</class>
 <widget class="QDialog" name="main_dialog">
  <property name="windowFlags">
   <set>Qt::CustomizeWindowHint|Qt::WindowTitleHint|Qt::WindowMinMaxButtonsHint|Qt::WindowCloseButtonHint</set>
  </property>
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EdPlanning</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="menu_widget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget{
       background-color: rgb(69, 69, 69, 220);
       outline: 0;
       }
       QListWidget::item {
       color: white;
       padding: 3px;
       }
       QListWidget::item::selected {
       color: black;
       background-color:palette(Window);
       padding-right: 0px;
       }
      </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>Catchment areas</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Settings</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>About</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="area_contents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>500</width>
         <height>450</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>450</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QStackedWidget" name="stacked_widget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="catchment_areas">
           <layout class="QVBoxLayout" name="vlayout_main">
            <item>
             <widget class="QgsCollapsibleGroupBox" name="mGroupBox_5589078">
              <property name="title">
               <string>Catchment area settings</string>
              </property>
              <layout class="QVBoxLayout" name="vlayout_content">
               <item>
                <layout class="QFormLayout" name="formlayout_config">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_pointlayer">
                   <property name="text">
                    <string>Point layer</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QgsMapLayerComboBox" name="combobox_layer"/>
                 </item>
                 <item row="1" column="1">
                  <widget class="QCheckBox" name="checkbox_selected_only">
                   <property name="text">
                    <string>Use only selected features</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_distance">
                   <property name="text">
                    <string>Distance</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QgsSpinBox" name="spinbox_distance">
                   <property name="minimum">
                    <number>5</number>
                   </property>
                   <property name="maximum">
                    <number>120</number>
                   </property>
                   <property name="singleStep">
                    <number>5</number>
                   </property>
                   <property name="value">
                    <number>30</number>
                   </property>
                   <property name="clearValue">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_buckets">
                   <property name="text">
                    <string>Distance divisions</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QgsSpinBox" name="spinbox_buckets">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>8</number>
                   </property>
                   <property name="singleStep">
                    <number>1</number>
                   </property>
                   <property name="value">
                    <number>1</number>
                   </property>
                   <property name="clearValue">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QGroupBox" name="groupbox_units">
                 <property name="title">
                  <string>Unit of distance</string>
                 </property>
                 <layout class="QHBoxLayout" name="hlayout_units">
                  <item>
                   <widget class="QRadioButton" name="radiobtn_mins">
                    <property name="text">
                     <string>Minutes</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radiobtn_meters">
                    <property name="text">
                     <string>Meters</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupbox_profile">
                 <property name="title">
                  <string>Profile</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <layout class="QGridLayout" name="glayout_profile">
                    <item row="0" column="0">
                     <widget class="QRadioButton" name="radiobtn_walking">
                      <property name="text">
                       <string>Walking</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QRadioButton" name="radiobtn_cycling">
                      <property name="text">
                       <string>Cycling</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QRadioButton" name="radiobtn_driving">
                      <property name="text">
                       <string>Driving</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="duration_label">
              <property name="text">
               <string></string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDialogButtonBox" name="buttonbox_main">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="standardButtons">
               <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="settings">
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <item>
             <widget class="QgsCollapsibleGroupBox" name="mGroupBox_4">
              <property name="title">
               <string>Graphhopper instance</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_gh_url">
                   <property name="text">
                    <string>GraphHopper URL</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="lineedit_url">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_gh_apikey">
                   <property name="text">
                    <string>API Key</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="lineedit_apikey">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QgsCollapsibleGroupBox" name="mGroupBox_20">
              <property name="title">
               <string>Result layer</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_20">
               <item>
                <layout class="QGridLayout" name="gridLayout_20">
                 <item row="0" column="0">
                  <widget class="QCheckBox" name="checkbox_file">
                   <property name="text">
                    <string>Write to directory</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QgsFileWidget" name="file_widget">
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QgsCollapsibleGroupBox" name="mGroupBox_3">
              <property name="title">
               <string>Logging</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>QGIS must be restarted in order to apply changes in log levels</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_level_console">
                   <property name="text">
                    <string>Console level</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QComboBox" name="combo_box_log_level_console"/>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_level_file">
                   <property name="text">
                    <string>File level</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="combo_box_log_level_file"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btn_open_log">
                   <property name="text">
                    <string>Open most recent log file</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="about">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Attributions</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QGridLayout" name="gridLayout_3">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetNoConstraint</enum>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_development">
                   <property name="text">
                    <string>Main development</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="label_gispo">
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://www.gispo.fi/en/home/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Gispo Ltd.&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                    </string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Sources</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_qsource">
                   <property name="text">
                    <string>Graphhopper repository</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="label_qgsource">
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/graphhopper/graphhopper&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://github.com/graphhopper/graphhopper&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                    </string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_gsource">
                   <property name="text">
                    <string>Edplanning Plugin repository</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="label_source">
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/GispoCoding/edplanning&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://github.com/GispoCoding/edplanning&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                    </string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Versions</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="1">
                  <widget class="QLabel" name="label_version">
                   <property name="text">
                    <string>&lt;placeholder&gt;</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_version_txt">
                   <property name="text">
                    <string>Edplanning Plugin</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Licenses</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_graphhopper">
                   <property name="text">
                    <string>Graphhopper</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="label_apache2">
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://www.apache.org/licenses/LICENSE-2.0&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Apache License 2.0&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                    </string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_plugin">
                   <property name="text">
                    <string>Edplanning Plugin</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="label_gpl3">
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://www.gnu.org/licenses/gpl-3.0.en.html&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;GNU General Public License v3.0&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                    </string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsExtentGroupBox</class>
   <extends>QgsCollapsibleGroupBox</extends>
   <header>qgsextentgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonbox_main</sender>
   <signal>accepted()</signal>
   <receiver>main_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonbox_main</sender>
   <signal>rejected()</signal>
   <receiver>main_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
